gettsim/data/soz_vers_beitr.yaml

```diff
@@ -167,12 +167,15 @@ gkvbs_an:
     2019-01-01:
       value: 0.082
       note:
+    2020-01-01:
+      value: 0.084
+      note: https://www.bundesgesundheitsministerium.de/presse/pressemitteilungen/2019/4-quartal/gkv-zusatzbeitragssatz.html
```

Comment:
Since 2019 the Parität is back. Now the additional contribution needs to be paid half by the employer and half by the employee.